docs/metrics.md

```diff
@@ -170,7 +441,7 @@ When defining a `histogram`, `buckets` must also be provided (see below).
 
 [Argo variables](variables.md) can be included anywhere in the metric spec, such as in `labels`, `name`, `help`, `when`, etc.
 
-Metric names can only contain alphanumeric characters, `_`, and `:`.
+Metric names can only contain alpha-numeric characters, `_`, and `:`.
```

Comment:
```suggestion
Metric names can only contain alphanumeric characters, `_`, and `:`.
```

"[alphanumeric](https://www.merriam-webster.com/dictionary/alphanumeric)" is a word with no hyphen